frontend/src/components/TheTeam.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import '../styles/TeamStyle.css';
+import img1 from '../assets/images/Bo_Mendez.jpg';
+import img2 from '../assets/images/Chelsea_Heredia.jpg';
+import img3 from '../assets/images/Tritian_Monis.png';
+import img4 from '../assets/images/Roshin_Lal.jpg';
+import img5 from '../assets/images/placement.png';
```

Comment:
Consider naming these imports something more meaningful as the name img1 does not tell me much.